Extension/src/nativeStrings.json

```diff
@@ -264,5 +264,133 @@
     "file_tag": "File",
     "compiler_default_language_standard_version_old" : "Compiler returned default language standard version: {0}. Since this version is old, will try to use newer version {1} as default.",
     "unexpected_output_from_clang_tidy": "Unexpected output from clang-tidy: {0}. Expected: {1}.",
-    "generate_doxygen_comment": "Generate Doxygen Comment"
+    "generate_doxygen_comment": "Generate Doxygen Comment",
+    "offer_create_declaration": {
+        "text": "Create declaration of {0} in {1}",
+        "hint": "{0} is the name of a C/C++ function, {1} is a file name."
+    },
+    "offer_create_definition": {
+        "text": "Create definition of {0} in {1}",
+        "hint": "{0} is the name of a C/C++ function, {1} is a file name."
+    },
+    "function_definition_not_found": {
+        "text": "Function definition for '{0}' not found.",
+        "hint": "{0} is the name of a C/C++ function."
+    },
+    "cm_attributes": {
+        "text": "Attributes",
+        "hint": "'Attributes' is a C++ specifier."
+    },
+    "cm_bases": {
+        "text": "Bases",
+        "hint": "'Bases' are a C++ class type."
+    },
+    "cm_classes": {
+        "text": "Classes",
+        "hint": "'Classes' are a C++ class type."
+    },
+    "cm_coclasses": {
+        "text": "CoClasses",
+        "hint": "'CoClasses' are a C++ class type."
+    },
+    "cm_delegates": {
+        "text": "Delegates",
+        "hint": "'Delegates' are a C++ delegate."
+    },
+    "cm_enums": {
+        "text": "Enums",
+        "hint": "'Enums' are a C++ enum."
+    },
+    "cm_events": {
+        "text": "Events",
+        "hint": "'Events' are a C++ event."
+    },
+    "cm_functions": {
+        "text": "Functions",
+        "hint": "'Functions' are a C++ function."
+    },
+    "cm_importdirectives": {
+        "text": "Import directives",
+        "hint": "'Import directives' are a C++ ."
```

Comment:
These hints don't seem correct. Can they be reworded?

Reply:
Those hints were place holders. I don't think these are used on the UI, but the internal API requests for them. Pending on what to do with these.